docs/cisco.asa.asa_acls_module.rst

```diff
@@ -2484,7 +2484,7 @@ Examples
     # access-list global_access line 4 extended deny tcp any any eq www log errors interval 300
     # access-list R1_traffic line 1 remark test_v6_acls
     # access-list R1_traffic line 2 extended deny tcp 2001:db8:0:3::/64 eq www 2001:fc8:0:4::/64 eq telnet inactive
-    # access-list temp_access line 1 extended deny tcp 192.0.2.0 255.255.255.0 192.0.3.0 255.255.255.0 eq www log default
+    # access-list temp_access line 1 extended deny tcp 192.0.2.0 255.255.255.0 198.51.100.0 255.255.255.0 eq www log default
     # access-list temp_access line 2 extended deny igrp 198.51.100.0 255.255.255.0 198.51.110.0 255.255.255.0
```

Comment:
So my guess is that this is not what we want. Do we need something larger than a /24 to make this work?

Reply:
```suggestion
    # access-list temp_access line 1 extended deny tcp 192.0.2.0 255.255.255.0 198.51.100.0 255.255.255.0 eq www log default
    # access-list temp_access line 2 extended deny igrp 198.51.100.0 255.255.255.0 203.0.113.0 255.255.255.0
```

Would this work out, sticking to the three documentation /24 networks?